### ISSUE TITLE
Remove unused columns from Events,Sponsors,Venues and Conference tables

### ISSUE THREAD

**Issue by nishanthvijayan** [openSUSE/osem#1013]
`logo_updated_at, logo_content_type, logo_file_size` columns in sponsors and conferences tables appear to be no longer needed (after #970)
Similarly 'photo_updated_at , photo_content_type, photo_file_size' columns in venues table
and `logo_updated_at, logo_content_type, logo_file_size, logo_file_name , progress, time_slots ` columns of events table also appear to be unused.
(not sure about progress and time_slots please verify)
Is there any reason why they still present? Shouldn't they be removed?

EDIT: and `use_difficulty_levels` in conferences
